packages/react-native-web/src/modules/ImageLoader/index.js

```diff
@@ -74,12 +74,13 @@ let id = 0;
 const requests = {};
 
 const ImageLoader = {
-  abort(requestId: number) {
-    let image = requests[`${requestId}`];
+  clear(requestId: number) {
+    const image = requests[`${requestId}`];
     if (image) {
       image.onerror = null;
       image.onload = null;
-      image = null;
+      ImageUriCache.remove(image.src);
+      image.src = '';
```

Comment:
renamed this to `clear` to represent more closely that it's not just for aborting but actually a necessary cleanup, as it also updates the `ImageUriCache` now

Existing functionality is unaffected (besides the rename)